rpm/nemo-qml-plugin-calendar-qt5.spec

```diff
@@ -47,3 +60,10 @@ rm -rf %{buildroot}
 %files tests
 %defattr(-,root,root,-)
 /opt/tests/nemo-qml-plugins-qt5/calendar/*
+
+%files lightweight
+%defattr(-,root,root,-)
+%attr(2755, root, privileged) %{_bindir}/calendardataservice
```

Comment:
Until we have a linkable library with proper devel headers this binary needs to be here. Notice that the binary is a helper to a QML plugin.